File: minimumWaitingTime/minimumWaitingTime.py
def minimumWaitingTime(queries):
    # Write your code here.
    queries.sort()
    totalWaitingTime = 0
    for i in range(len(queries)):
        duration = queries[i]
        queriesLeft = len(queries) - (i + 1)
        totalWaitingTime += duration * queriesLeft
        print(totalWaitingTime,  duration, queriesLeft )
    return totalWaitingTime

queries = [3, 2, 1, 2, 6]
print(minimumWaitingTime(queries))

File: minimumWaitingTime/minimumWaitingTime.js
function minimumWaitingTime(queries) {
    let sorted_q = queries.sort((a, b) => {return a - b})
    console.log(sorted_q);

    let waitingTime = 0;
    queries.forEach((num) =>{
        console.log(num);
    })
    return waitingTime;
}

let queries = [3, 2, 1, 2, 6];
console.log(minimumWaitingTime(queries));